packages/sit-onyx/src/components/examples/GridPlayground/GridPlayground.vue

```diff
@@ -1,167 +1,275 @@
-    <main
-      ref="gridElement"
+  <div class="onyx-text playground">
+    <div class="onyx-grid-container">
+      <OnyxHeadline is="h1" class="playground__headline">Grid and breakpoint demo</OnyxHeadline>
+      <OnyxHeadline is="h2">
+        Your current breakpoint:
+        <span class="playground__breakpoint">{{ currentBreakpoint }}</span>
+      </OnyxHeadline>
+
+      <p>
+        Add placeholder components below and link them to the grid properties to get an
+        understanding on how components will behave inside the grid. To see the responsiveness of
+        the onyx grid in action, just use the window resizer to adjust your browser width.
+      </p>
+
+      <p>
+        For further details on the grid, please refer to our
+        <OnyxLink href="https://onyx.schwarz/development/grid.html" target="_blank"
+          >grid docs</OnyxLink
+        >
+      </p>
+
+      <div class="playground__options">
+        <OnyxRadioGroup
+          v-model="gridSettings.maxWidth"
+          headline="Max content width"
+          :options="maxWidthOptions"
+          direction="horizontal"
+        />
+
+        <OnyxRadioGroup
+          v-model="gridSettings.alignment"
+          headline="Alignment"
+          :options="alignmentOptions"
+          direction="horizontal"
+          :disabled="gridSettings.maxWidth === 'none'"
+        />
+
+        <OnyxRadioGroup
+          v-if="viewportSize.width.value >= ONYX_BREAKPOINTS.xl"
+          v-model="gridSettings.maxColumns"
+          headline="Max columns"
+          :options="maxColumnsOptions"
+          direction="horizontal"
+        />
+      </div>
+
+      <div v-if="gridValues" class="playground__badges">
+        <GridBadge :value="gridValues.margin" label="Margin" color="danger" />
+        <GridBadge :value="gridValues.columnCount" label="Columns" color="warning" />
+        <GridBadge :value="gridValues.gutter" label="Gutter" color="info" />
+      </div>
+    </div>
+
+    <div
       :class="{
-        'onyx-grid': true,
-        'onyx-grid-container': true,
-        'onyx-grid-center': isCentered,
-        'onyx-grid-xl-20': is20Xl,
-        'onyx-grid-max-md': isMaxMd,
-        'onyx-grid-max-lg': isMaxLg,
+        'onyx-grid-center': gridSettings.alignment === 'center',
+        [`onyx-grid-max-${gridSettings.maxWidth}`]: gridSettings.maxWidth !== 'none',
+        'onyx-grid-xl-20': gridSettings.maxColumns === 20,
       }"
     >
-      <GridElement
-        v-for="(element, i) in elements"
-        :key="i"
-        :settings="element"
-        :selected="element === selectedElement"
-        @click="selectedElement = elements[i]"
-      />
-
-      <button
-        type="button"
-        class="onyx-grid-playground__add-new"
-        @click="elements.push([{ spans: 4 }])"
-      >
-        +
-      </button>
-    </main>
+      <GridOverlay :columns="gridValues?.columnCount" />
+
+      <div>
+        <OnyxNavBar app-name="Example navigation">
```

Comment:
could be misleading, as this represents the app-name. maybe use something like
```suggestion
        <OnyxNavBar app-name="Demo App">
```
?